src/components/AppSidebar/AppSidebar.vue

```diff
@@ -123,38 +123,42 @@
 				</div>
 
 				<!-- sidebar details -->
-				<div v-if="!empty" :class="{ 'app-sidebar-header__desc--with-star': canStar, 'app-sidebar-header__desc--with-subtitle': subtitle && !titleEditable, 'app-sidebar-header__desc--editable': titleEditable && !subtitle, 'app-sidebar-header__desc--with-subtitle--editable': titleEditable && subtitle}" class="app-sidebar-header__desc">
+				<div v-if="!empty" :class="{ 'app-sidebar-header__desc--with-star': canStar, 'app-sidebar-header__desc--with-subtitle': subtitle && !titleEditing, 'app-sidebar-header__desc--editable': titleEditing && !subtitle, 'app-sidebar-header__desc--with-subtitle--editable': titleEditing && subtitle}" class="app-sidebar-header__desc">
```

Comment:
Let's split classes to individual lines please :stuck_out_tongue:

Reply:
Done.